--- ltmodem-8.26a9/source/configure.in	2002-11-25 03:09:39.000000000 +0100
+++ ltmodem-8.26a9.pius/source/configure.in	2003-09-18 17:30:57.000000000 +0200
@@ -134,7 +134,6 @@
 
 echo "Checking kernel header version"
 KVERSION="conftest.kversion"
-CFLAGS="-I$KERNEL_SOURCE_TREE/include"
 AC_TRY_RUN(
 #include "$KERNEL_SOURCE_TREE/include/linux/autoconf.h"
 #include "$KERNEL_SOURCE_TREE/include/linux/config.h"
--- ltmodem-8.26a9/source/Makefile.in	2002-11-18 17:04:10.000000000 +0100
+++ ltmodem-8.26a9.pius/source/Makefile.in	2003-09-18 17:30:07.000000000 +0200
@@ -14,7 +14,7 @@
 
 ifeq ($(GCCVERSION),3)
 KCFLAGS = -D__KERNEL__ -I@KERNEL_SOURCE_TREE@/include \
-        -Wall -Wstrict-prototypes -O3 \
+        -Wall -Wstrict-prototypes -O3 \
         -fomit-frame-pointer -fno-strict-aliasing -pipe \
         -fno-strength-reduce \
         -DMODULE -DLT_VERSION=$(LT_VERSION) -DLINUX \
@@ -22,9 +22,9 @@
         -DLT_SERIAL_MODULE=$(subst lt_,lt$(LTNUMBER)_,@LT_SERIAL_MODULE@)
 else
 KCFLAGS = -D__KERNEL__ -I@KERNEL_SOURCE_TREE@/include \
-        -Wall -Wstrict-prototypes -O3 \
+        -Wall -Wstrict-prototypes -O3 \
         -fomit-frame-pointer -fno-strict-aliasing -pipe \
-        -fno-strength-reduce -m486 -malign-loops=2 \
+        -fno-strength-reduce -malign-loops=2 \
         -malign-jumps=2 -malign-functions=2 -DCPU=586 \
         -DMODULE -DLT_VERSION=$(LT_VERSION) -DLINUX \
         -DNO_EEPROM \
