diff -Nur ltmodem-5.99b.orig/source/Makefile.in ltmodem-5.99b/source/Makefile.in
--- ltmodem-5.99b.orig/source/Makefile.in	Fri May 18 20:54:06 2001
+++ ltmodem-5.99b/source/Makefile.in	Mon May 21 22:49:13 2001
@@ -9,9 +9,9 @@
 LT_PROPRIETARY_MODULE = @LT_PROPRIETARY_MODULE@.o
 
 KCFLAGS = -D__KERNEL__ -I@KERNEL_SOURCE_TREE@/include \
-	-Wall -Wstrict-prototypes -O3 \
+	-Wall -Wstrict-prototypes \
 	-fomit-frame-pointer -fno-strict-aliasing -pipe \
-	-fno-strength-reduce -m486 -malign-loops=2 \
+	@CFLAGS@ -malign-loops=2 \
 	-malign-jumps=2 -malign-functions=2 -DCPU=586 \
 	-DMODULE -DLT_VERSION=$(LT_VERSION) -DLINUX \
 	-DNO_EEPROM -DLT_SERIAL_MODULE=@LT_SERIAL_MODULE@
